src/pro/sky/java/course1/Employee.java

```diff
@@ -0,0 +1,55 @@
+package pro.sky.java.course1;
+
+public class Employee {
+    static int cnt = 0;
+    private int id;
+    private String fio;
+    private int oldel;
```

Comment:
опечатка
```suggestion
    private int otdel;
```
лучше использовать явные английские слова: department